data/in/legislature/Dr-Tyler-Johnson-5e19b0fa-7bdc-4732-826c-adf5fbad4e21.yml

```diff
@@ -0,0 +1,28 @@
+id: ocd-person/5e19b0fa-7bdc-4732-826c-adf5fbad4e21
+name: Dr. Tyler Johnson
+email: Senator.Johnson@iga.in.gov
+image: https://cdn.zephyrcms.com/91785bff-d689-4351-a1fa-2345256bf313/-/progressive/yes/-/format/jpeg/-/scale_crop/200x300/center/-/stretch/off/johnson-website.jpg
+party:
+- name: Republican
+roles:
```

Comment:
@sroomf is it ok that their "name" is Dr. Tyler Johnson. I'm fine with it but didn't know if it would cause issues.

Reply:
hmm...good point. i'll change it! i think it won't cause issues